src/test/scala/sigmastate/CostingSpecification.scala

```diff
@@ -202,4 +202,10 @@ class CostingSpecification extends SigmaTestingData {
     cost(s"{ $selfTree.get(key1, lookupProof).isDefined }") shouldBe (AccessTree + containsCost + constCost + selectField)
     cost(s"{ $selfTree.getMany(keys, lookupProof).size > 0 }") shouldBe (AccessTree + containsCost + constCost + LengthGTConstCost)
   }
+
+  property("Coll operations cost") {
+    val coll = "OUTPUTS"
+    cost(s"{ $coll.filter({ (b: Box) => b.value > 1 }).size > 0 }") shouldBe
+      (OutputsCost + collToColl + lambdaCost + extractCost + GTConstCost + LengthGTCost)
```

Comment:
@aslesarenko I'm having hard times figuring out our costing (mine: 73, actual: 90). Is there any technique to make it easier? I've traced calls to CostTable, but it was not enough to figure out what I'm missing in the expected cost.

Reply:
@greenhat you can check the graphs generated and saved in file.
This is how the file name is specified
```  
    val env: ScriptEnv = Map(
    ScriptNameProp -> s"filename_verify",
  ...
```
The file should be in `test-out` directory.
The graph should have explicit nodes like `CostOf(...)`, which represent access to CostTable entries.
The actual cost is counted in the nodes like this `s1340: Int = OpCost(2, List(s1361, s1360), s983)`
Each such node is handled like
`costAccumulator.add(s1340, OpCost(2, List(s1361, s1360), s983), dataEnv)`
See `CostAccumulator`

How much cost is represented by OpCost node?
1) Symbols s1361, s1360 are dependencies. They represent cost that should be accumulated before s983. 
2) If upon handling of OpCost, the dependencies are not yet accumulated, then they are accumulated first, and then s983 is accumulated.
3) the values of s1340 is the value of s983.
4) Thus execution of OpCost, consists of 2 parts: a) data flow b) side effect on CostAccumulator
5) OpCost is special node, interpreted in a special way. See method evaluate in Evaluation.